envoy-control-source-consul/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/consul/ConsulProperties.kt

```diff
@@ -15,7 +15,8 @@ class ConsulProperties {
 class ConsulWatcherOkHttpProperties {
     var readTimeout: Duration = Duration.ofMinutes(6)
     var connectTimeout: Duration = Duration.ofSeconds(2)
-    var maxRequests = 1000
+    // TODO: is it even valid to have maxRequests != dispatcherMaxPoolSize? Maybe we should reduce it to one property
```

Comment:
Let me know what you think about this

Reply:
I don't remember why we have it separately. Separation doesn't make sense in our context IMO, so yeah, I would reduce it 👍.

Reply:
👍

Reply:
done: max-requests removed